juriscraper/opinions/united_states/federal_appellate/ca1.py

```diff
@@ -1,124 +1,98 @@
 
-    def _get_case_names(self):
-        return [
-            e.strip()
-            for e in self.html.xpath(
-                "//tr[position() > 1]/td[4]/text()[contains(., 'v.')]"
+    def _process_html(self):
+        for row in self.html.xpath("//tr[position() > 1]"):
+            title = row.xpath("td[2]/a/text()")[0]
+            url = row.xpath("td[2]/a/@href")[0]
+            status = self.get_status_from_opinion_title(title)
+            docket = row.xpath("td[3]/a/text()")[0]
+            date_filed = row.xpath("td[1]/span/text()")[0]
+            name = row.xpath("td[4]/text()")[0]
+            lower_court = row.xpath("td[4]/span/text()")[0]
+            self.cases.append(
+                {
+                    "name": name.strip(),
+                    "url": url,
+                    "date": date_filed,
+                    "status": status,
+                    "docket": docket,
+                    "lower_court": lower_court,
+                }
             )
-        ]
-
-    def _get_download_urls(self):
-        return [
-            e for e in self.html.xpath("//tr[position() > 1]/td[2]//@href")
-        ]
-
-    def _get_case_dates(self):
-        dates = []
-        for s in self.html.xpath("//tr[position() > 1]/td[1]//text()"):
-            s = s.replace(r"\t", "").replace(r"\n", "").strip()
-            if s == "1996/05/32":
-                s = "1996/05/30"  # My life is thus lain to waste.
-            dates.append(datetime.strptime(s.strip(), "%Y/%m/%d").date())
-        return dates
 
-    def _get_docket_numbers(self):
-        regex = re.compile(r"(\d{2}-.*?\W)(.*)$")
-        docket_numbers = []
-        for s in self.html.xpath("//tr[position() > 1]/td[2]/a/text()"):
-            s = s.replace("O1-", "01-")  # I grow older, the input grows worse.
-            docket_numbers.append(
-                regex.search(s).group(1).strip().replace(".", "")
-            )
-        return docket_numbers
+    def get_status_from_opinion_title(self, title: str) -> str:
+        """Status is encoded in opinion's link title
 
-    def _get_precedential_statuses(self):
-        statuses = []
-        for text in self.html.xpath("//tr[position() > 1]/td[2]//@href"):
-            if "U" in text:
-                statuses.append("Unpublished")
-            elif "P" in text:
-                statuses.append("Published")
-            elif "E" in text:
-                statuses.append("Errata")
-            else:
-                statuses.append("Unknown")
-        return statuses
+        :param title: opinion title. Ex: 23-1667P.01A, 23-1639U.01A
 
-    def _get_lower_courts(self):
-        lower_courts = []
-        for e in self.html.xpath("//tr[position() > 1]/td[4]/font"):
-            try:
-                lower_courts.append(e.xpath("./text()")[0].strip())
-            except IndexError:
-                lower_courts.append("")
-        return lower_courts
+        :return: status string
+        """
+        status = ""
```

Comment:
this line is unused

Reply:
Good eye, Just updated the PR removing this